src/Components/Publishing/Series/Components/SeriesAbout.tsx

```diff
@@ -0,0 +1,112 @@
+import React, { Component  } from "react"
+import styled, { StyledFunction } from "styled-components"
+import { pMedia } from "../../../Helpers"
+import { Fonts } from "../../Fonts"
+import { PartnerBlock } from '../../Partner/PartnerBlock'
+import { Text } from '../../Sections/Text'
+
+interface Props {
+  color?: string,
+  series?: any
+}
+
+export class SeriesAbout extends Component<Props, null> {
+  public static defaultProps: Partial<Props>
+
+  render () {
+    const { color, series } = this.props
+    const { about, sponsor } = series
```

Comment:
This `series` is the same thing as `article`, right? Do you think that could be confusing? I think if we end up modeling Series as an entirely different model, this would make sense but in this implementation we should probably call it what it is. I'm thinking of it as a different article _layout_ here. But I admit there's still some ambiguity overall on how we want to approach this. Maybe we can leave this as is for now and loop back sometime tomorrow on a full plan for modeling Series.